static/app/views/profiling/landing/profileCharts.tsx

```diff
@@ -108,21 +118,26 @@ export function ProfileCharts({query, router, selection}: ProfileChartsProps) {
               },
               {
                 top: '32px',
-                left: '52%',
+                left: hideCount ? '24px' : '52%',
```

Comment:
this is kind of a lazy hack... 

we can remove all the code paths that adds the `count()` series and supplies it to the `<AreaChart />` which requires changing many props... or we can just visually hide the first chart, which is what we're doing here.

Reply:
Those two charts probably should have been split in the first place and just receive data as props, but this works